airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -123,14 +123,10 @@ def get_access_token(config: Mapping[str, Any]):
     def _get_authenticator(self, config: Mapping[str, Any]):
         _, token = self.get_access_token(config)
         tokens = [t.strip() for t in token.split(constants.TOKEN_SEPARATOR)]
-        requests_per_hour = config.get("requests_per_hour")
-        if requests_per_hour:
-            return MultipleTokenAuthenticatorWithRateLimiter(
-                tokens=tokens,
-                auth_method="token",
-                requests_per_hour=requests_per_hour,
-            )
-        return MultipleTokenAuthenticator(tokens=tokens, auth_method="token")
+        return MultipleTokenAuthenticatorWithRateLimiter(
+            tokens=tokens,
+            auth_method="token",
```

Comment:
Is there a complete need to provide the auth_method here? Can we store it inside the superclass?